templates/includes/records/archive-collections.html

```diff
@@ -3,7 +3,7 @@
         <h2>Collections information</h2>
             {% for collection in record.archive_collections.collection_info_list %}
                 {% if collection.count > 0 %}
-                    <h3>{{ collection.long_display_name }}({{ collection.count }})</h3>                    
+                    <h3  id="heading-{{collection.name}}">{{ collection.long_display_name }}({{ collection.count }})</h3>                    
```

Comment:
@TNA-Allan Any chance of getting a space after the `long_display_name` (as suggested previously)?

Reply:
Sorry I missed it out. Done now.